src/components/panel/panel.spec.js

```diff
@@ -91,6 +91,52 @@ describe('$mdPanel', function() {
     expect(PANEL_WRAPPER_CLASS).not.toHaveClass(HIDDEN_CLASS);
   });
 
+  it('should trap focus inside of the panel with trapFocus=true', function() {
```

Comment:
Should these tests be under the config options describe block? Or as a separate describe for a11y?